src/OmniSharp.Abstractions/Models/v2/CodeStructure/CodeElement.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+
+namespace OmniSharp.Models.V2.CodeStructure
+{
+    public partial class CodeElement
+    {
+        public string Kind { get; }
+        public string Name { get; }
+        public string DisplayName { get; }
+        public IReadOnlyList<CodeElement> Children { get; }
+        public IReadOnlyDictionary<string, Range> Ranges { get; }
```

Comment:
I just noticed this while working on BlockStructure--seems like the Point type has a "ZeroBasedIndexJsonConverter" on its fields which results in the serialized values from OmniSharp being one-based instead of zero-based. Very annoying if you're consuming in C# extension...

Reply:
Yes, but that's the right thing. OmniSharp is one-based by default so we should keep it consistent. There's an option that can passed to OmniSharp to make everything zero-based. We should probably just invest in C# for VS Code to convert it over to using OmniSharp in a zero-based fashion.

Reply:
Ah, good to know. Thanks!

Reply:
Yeah, here's the flag: https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.Host/CommandLineApplication.cs#L34

We can pass that in C# for VS Code when we launch OmniSharp, but we'll need to update a ton of places in VS Code where we convert from one-based.

Reply:
Indeed the original editors vim/emacs were all one based, hence that's how OmniSharp has always behaved.  I added the converter to make it easier to write against Roslyn but lets OmniSharp respect the editors design.

Reply:
Yup! Well said @david-driscoll